src/core/public/doc_links/doc_links_service.ts

```diff
@@ -193,8 +198,11 @@ export class DocLinksService {
           alertsKibanaDiskThreshold: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/kibana-alerts.html#kibana-alerts-disk-usage-threshold`,
           alertsKibanaJvmThreshold: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/kibana-alerts.html#kibana-alerts-jvm-memory-threshold`,
           alertsKibanaMissingData: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/kibana-alerts.html#kibana-alerts-missing-monitoring-data`,
+          metricbeatBlog: `${ELASTIC_WEBSITE_URL}blog/external-collection-for-elastic-stack-monitoring-is-now-available-via-metricbeat`,
```

Comment:
nit: As this URL doesn't rely on `${DOC_LINK_VERSION}`, and there might be more metricbeat blog links in the future, it might make more sense to keep the prior implementation where the consuming code is just using `ELASTIC_WEBSITE_URL` directly with the blog path. Or alternatively to export a `ELASTIC_WEBSITE_BLOG_URL` to be used with a corresponding path. But no strong feelings on that.

Reply:
So far I haven't found a lot of blog links, so I think this announcement was exceptional. If more of them get added in the future, I agree that we should consider having a different base url that all of them can share (though I'd still vote for having keywords in the shared file rather than having the blog URLs sprinkled across the code.. we don't test those so it's easy to lose track of them).